catalog/rest/rest_test.go

```diff
@@ -445,6 +445,60 @@ func (r *RestCatalogSuite) TestCreateNamespace200() {
 	r.Require().NoError(cat.CreateNamespace(context.Background(), catalog.ToIdentifier("leden"), nil))
 }
 
+func (r *RestCatalogSuite) TestCheckNamespaceExists204() {
+	r.mux.HandleFunc("/v1/namespaces/leden", func(w http.ResponseWriter, req *http.Request) {
+		r.Require().Equal(http.MethodGet, req.Method)
```

Comment:
should be a HEAD request

https://github.com/apache/iceberg/blob/2fa6cd855b4e43b383df7b72776eb554b60d8c06/open-api/rest-catalog-open-api.yaml#L376-L406

Reply:
oh i see, youre using the load namespace API
https://github.com/apache/iceberg/blob/2fa6cd855b4e43b383df7b72776eb554b60d8c06/open-api/rest-catalog-open-api.yaml#L340-L374

Reply:
Hi @kevinjqliu , thanks so much for the sharp eyes. I see that from the OpenAPI doc its a method HEAD. I have modified the `CheckNamespaceExists` method to use `doHead` instead of `doGet` method. :pray: